tests/test_files/calculator/calculator.py

```diff
@@ -1,5 +1,24 @@
-# TODO: Implement a calculator program, that can add, subtract, multiply and divide two numbers.
```

Comment:
Please delete the unnescesary TODO comments. Try to write docstring. You could find an example in argument parser.